src/Analyzers/CSharp/Tests/RemoveRedundantEquality/RemoveRedundantEqualityTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.RemoveRedundantEquality;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Testing;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+using VerifyCS = CSharpCodeFixVerifier<
+    CSharpRemoveRedundantEqualityDiagnosticAnalyzer,
+    RemoveRedundantEqualityCodeFixProvider>;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.RemoveRedundantEquality
+{
+
+    public class RemoveRedundantEqualityTests
```

Comment:
Currently this lacks the following tests: (I'll add them soon):

- Testing when the "True" or "False" part.
- Testing when one side is `Nullable<bool>` (`bool?`)